app/components/chart-footnotes.tsx

```diff
@@ -95,28 +96,40 @@ export const ChartFootnotes = ({
   const sparqlEditorUrl =
     visibleData?.dataCubeByIri?.observations.sparqlEditorUrl;
 
+  const formatLocale = useTimeFormatLocale();
   const cubeLink = useMemo(() => {
     return makeOpenDataLink(locale, data?.dataCubeByIri);
   }, [locale, data?.dataCubeByIri]);
 
   if (data?.dataCubeByIri) {
     const { dataCubeByIri } = data;
 
+    const titleContent = `${dataCubeByIri.title}${
+      dataCubeByIri.dateModified
+        ? t({
+            id: "dataset.metadata.date.footnotes.updated",
+            message: `, latest update: ${formatLocale.format("%x %X")(
```

Comment:
TIL about `%x` time formats 📆

Reply:
I changed the %X to %H:%M otherwise we have seconds in the output and this does not seem helpful. There is no equivalent of %X without seconds https://github.com/d3/d3-time-format/issues/34.